gc/base/PhysicalSubArenaRegionBased.hpp

```diff
@@ -107,7 +107,6 @@ class MM_PhysicalSubArenaRegionBased: public MM_PhysicalSubArena
 	 * @param env[in] the current thread
 	 */
 	void validateNumaSymmetry(MM_EnvironmentBase *env);
-
```

Comment:
I would exclude changes in this file from this pull request

Reply:
done